### ISSUE TITLE
Form Field Description - 📝 Acceptatiecriteria voor accessibility

### ISSUE THREAD

**Issue by RenateRoke** [nl-design-system/candidate#946]
# 📝 Acceptatiecriteria voor accessibility

Het is belangrijk dat het duidelijk is op welke manier de Candidate component aan toegankelijkheidseisen moet voldoen en met welke succescriteria rekening moet worden gehouden bij het gebruiken van de component in context.

- [x] Bekijk community implementaties
- [x] Analyseer auditrapporten
- [x] Stel acceptatiecriteria voor de component zelf vast
- [x] Stel acceptatiecriteria vast voor de component in context
- [ ] Maak een PR in de documentatie repository
- [x] Maak een voorstel voor een toegankelijke HTML Semantiek

zie [verzameldocument](https://codefornl.slack.com/docs/T68FXPFQV/F09TNA2S01Y)

## Bekijk community implementaties

Bekijk welke implementaties er al zijn in de community en beoordeel welke opties daarvan (of helemaal niet) toegankelijke opties zijn.

## Analyseer auditrapporten

Analyseer welke toegankelijkheidsproblemen vaak voorkomen bij de component, zowel in de component zelf, als door de context. Het makkelijkst is de notities in Archimate bekijken over de component, voor zover de voorraad strekt. Die zijn gebaseerd op honderden auditrapporten. https://architectuur-tmp.vercel.app

## Stel acceptatiecriteria voor de component zelf vast

Stel aan de hand van de onderzoeksstappen de acceptatiecriteria van de component zelf vast. Waar (wcag succescriteria) voldoet het kale component aan als de component candidate en later hall of fame zijn. Dit zijn dus alleen de succescriteria die direct te hebben met interne semantiek/rollen/attributen (en basis thema css), zonder cóntext van omliggende componenten op een pagina.

## Stel acceptatiecriteria vast voor de component in context

Stel aan de hand van de onderzoeksstappen nu de acceptatiecriteria vast van de component in context met omloggende componenten op een pagina. Verdeel deze criteria die wèl contextgerelateerd zijn onder in Developer, Designer, contentmaker

## Maak een PR in de documentatie repository

Voeg de beide categorieën toe als toegankelijkheids-acceptatiecriteria op de componentpagina. Vergeet niet te verwijzen naar relevante richtlijnen. Dit komt later in de publicatiefase online.

## Maak een voorstel voor een toegankelijke HTML Semantiek

Maak een voorstel voor HTML semantiek-opties die zouden voldoen aan de opgestelde acceptatiecriteria voor de component en stem dit af met de Developer die de React component maakt. Voeg ook eventuele dingen toe die voor design belangrijk zijn en deel die met de Designers.

🚩 Checkpoint
📝 Acceptatiecriteria a11y
